robotbase/src/main/java/frc/robot/subsystems/CommandSwerveDrivetrain.java

```diff
@@ -128,10 +136,32 @@ public Consumer<ChassisSpeeds> getChassisSpeedsConsumer() {
       public void accept(ChassisSpeeds speeds) {
         SwerveModuleState[] moduleStates = getKinematics().toSwerveModuleStates(speeds);
         for (SwerveModuleState state : moduleStates) {
-          state.speedMetersPerSecond += SWERVE.STATIC_FEEDFORWARD_METERS_PER_SECOND;
+          state.speedMetersPerSecond += Constants.SWERVE.STATIC_FEEDFORWARD_METERS_PER_SECOND;
         }
         setControl(chassisSpeedRequest.withSpeeds(getKinematics().toChassisSpeeds(moduleStates)));
       }
     };
   }
+
+  public ChassisSpeeds getChassisSpeeds() {
+    ChassisSpeeds chassisSpeeds = getKinematics().toChassisSpeeds(getModuleStates());
+    return chassisSpeeds;
+  }
+
+  public double getTargetLockRotations() {
```

Comment:
Looks like you are not calling this anymore. It looks like you had some merge issues.

Reply:
Fixed